toolchain/check/context.h

```diff
@@ -157,6 +157,11 @@ class Context {
     return import_ir_constant_values_;
   }
 
+  auto cpp_carbon_file_locations()
+      -> llvm::SmallVector<clang::SourceLocation, 0>& {
```

Comment:
```suggestion
      -> llvm::SmallVector<clang::SourceLocation>& {
```
to match the other suggestion.